script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt

```diff
@@ -176,37 +175,51 @@ Not only does it fail to penetrate her skin,
 [sha:606aa2997a4936ed351792e9b6891f7975a0ba05]{
 -- Page 207, Offset 40018.
 -- 『“肘なら脆いはず、なんて笑わせないで。
+%{i}
 "You thought my elbow was a weak point? Please.
+%{/i}
 }
 [sha:8daca244495ebc66914e19b152e6008589cc5e5e]{
 -- Page 207, Offset 40019.
 -- どこだろうと肌の硬さは同じものでしょう。
-My skin is the same hardness all over.
+%{i}
+Why wouldn't my skin be the same hardness all over?
+%{/i}
 }
 [sha:7df256842d01366f04f40062e4cd8e80f14b0d46]{
 -- Page 207, Offset 40020.
 -- そんなコトも分からなかったの？”』
-Did you not even realize that simple fact?"
+%{i}
+Did you not even realize that %{/i}obvious%{i} fact?"
+%{/i}
 }
 [sha:6c567d3539d424a006b9b8047a4acb5f8e5f9223]{
 -- Page 208, Offset 40021.
 -- 『“おつかれさま。
+%{i}
 "You never fail to amaze.
+%{/i}
```

Comment:
This one is a bit weird, but I'm not sure how to render it

Reply:
I think this is a sort of sarcastic "well done", but aside from literally just that, I don't know if there's a better way of rendering it

Reply:
"At least you tried." ?

![060](https://user-images.githubusercontent.com/2476799/236358690-7707aeaa-9655-40a4-affb-7645e2af366f.png)

Reply:
Let's go

Reply:
Is that a "sounds good, ship it"?